### ISSUE TITLE
[1.12.2] Forestry leaves are absolutelly black color

### ISSUE THREAD

**Issue by KorDum** [TeamJM/journeymap#22]
#### JourneyMap Version:

`1.12.2-5.6.0b2` or below

#### Forge Version:

`1.12.2-14.23.1.2590`

#### Describe the problem in repeatable steps:

1. grow any Forestry tree
2. open the map
3. Forestry leaves are absolutelly black color

**Comment by MauveCloud** [comment]
I ran across this myself today, and I can add a few clarifications:
1. player-planted Forestry trees have black leaves on the map and identify as "ForestryLeaves" when I hover over them in the fullscreen map, but worldgen Forestry trees have grayish leaves on the map and identify as blank when I hover over them.
2. I tried using the vanilla "map" item, and the leaves showed up normally there, so this is unlikely to be Forestry's fault.
